.github/workflows/tekton_task_tests.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           files: |
             tasks/**
+            internal/tasks/**
           dir_names: "true"
           dir_names_max_depth: "4"
```

Comment:
I found a bug here. 

I was wondering why we had this here. When this was added, we would have "catalog/task/..." but even then "3" would make more sense than "4". I think the idea was to only list the task dirs and not the tests dirs.

From the [documentation](https://github.com/tj-actions/changed-files):
```
    # Limit the directory output to 
    # a maximum depth e.g `test/test1/test2` 
    # with max depth of `2` 
    # returns `test/test1`. 
    # Type: string
    dir_names_max_depth: ''
```

When you change both a task and a test, you would end up with e.g. `tasks/create-pyxis-image/tests tasks/create-pyxis-image` and the tests would run twice. You can see it happen here in my latest PR: https://github.com/konflux-ci/release-service-catalog/actions/runs/10456434922/job/28953728848

So I think setting `dir_names_max_depth: "2"` would fix it now, but not for the new `internal` dir.

I was trying to come up with a simple script to remove the "duplicates" before calling the test scripts, but it seemed pretty cumbersome. I think the easiest solution would be to have two changed-files steps - one for `tasks/**` (with `dir_names_max_depth: "2") and the other for `internal/tasks/**` (with `dir_names_max_depth: "3"). And then `TEST_ITEMS: ${{ steps.changed-dirs.outputs.all_changed_files }} ${{ steps.changed-dirs-internal.outputs.all_changed_files }}` in the test tasks step. WDYT?

Reply:
So what happens if I only change a test file and not the task? Wouldn't it not be fetched?

Reply:
It will still return the parent dir (the nearest with the max depth). At least that's how I read the documentation. And it should be easy to test if it holds true (an extra commit with a small change inside a test that you remove afterwards).

Reply:
updated and tested

Reply:
Awesome, thanks!